packages/eslint-plugin-mdx/README.md

```diff
@@ -223,6 +137,8 @@ If you want to disable or change severity of some related rules, it won't work b
 }
 ```
 
+Some plugins are ESM and eslint don't supports them. So, a workaround is to set `ignoreRemarkConfig` to `true` and execute `remark-lint` through the terminal before running eslint. For example: `remark **/*.mdx --no-stdout && eslint . --fix --ext .mdx`.
```

Comment:
```suggestion
Some plugins are ESM and eslint don't supports them. So, a workaround is to set `ignoreRemarkConfig` to `true` and execute `remark-lint` through the terminal before running eslint. For example: `remark . --output --use mdx --ext .mdx && eslint . --fix --ext .mdx`.
```

How about this? This really uses remark and remark-mdx on the MDX files?

Reply:
Oh, I think this can be removed, we have been supporting ESM remark plugins now.